examples/angular_bazel_architect/BUILD.bazel

```diff
@@ -9,8 +14,7 @@ architect(
     configuration_env_vars = ["NG_BUILD_CACHE"],
     data = glob(
         [
-            "src/*",
-            "src/**",
+            "src/**/*"
```

Comment:
note these are not equivalent - starlark globbing `**` doesn't match zero segments

Reply:
Oh good to know that.

Reply:
@alexeagle I double check the docs again and it seems that `**` does match zero segments

From https://docs.bazel.build/versions/master/be/functions.html#glob
> **/bar/**/*.txt matches every .txt file in every subdirectory of this package, if at least one directory on the resulting path is called bar, such as xxx/bar/yyy/zzz/a.txt or bar/a.txt (remember that ** also matches zero segments) or bar/zzz/a.txt